### PR TITLE
Migrate test helpers to TypeScript

### PR DESCRIPTION
**Pull request by andreyfel**

### CODE REVIEW

File: ember-power-select/src/test-support/helpers.ts [hunk 1]
```diff
@@ -50,15 +59,20 @@ export function typeInSearch(scopeOrText, text) {
   return fillIn(selectors, text);
 }
 
-export async function clickTrigger(scope, options = {}) {
+export async function clickTrigger(
+  scope: string,
```
Comment (mkszepp) [inline ember-power-select/src/test-support/helpers.ts:63]: i think, scope should be optional or?

**Review by mkszepp — APPROVED** [review]
LGT, thank you so much for all changes

**Comment by andreyfel** [comment]
@mkszepp  Thanks for merging! Do you mind cutting a new release with these changes?

**Comment by mkszepp** [comment]
@andreyfel yes we can do it.. do you need it asap (because its blocking you for updating to v8) or its okay in next days?

**Comment by andreyfel** [comment]
@mkszepp it is blocking updating to v8 although it is not urgent and obviously can wait till Monday as the previous version still works :)
Thank you for the work you've put in to modernize this addon and make it ready for the future Ember!

**Comment by mkszepp** [comment]
released in `v8.1.0`

**Comment by andreyfel** [comment]
@mkszepp Thanks a lot!

**Pull request merged**